### PR TITLE
feat(@vtmn/css): add small size for `VtmnTag` component

### PR DESCRIPTION
**Pull request by EGilleron**
## Changes description
Link of the issue : https://github.com/Decathlon/vitamin-web/issues/1319

Small : 
image

Medium : 
image

### CODE REVIEW

File: packages/sources/css/src/components/indicators/tag/src/index.css [hunk 1]
```diff
@@ -94,3 +94,19 @@ a.vtmn-tag:focus-visible {
   background-color: var(--vtmn-semantic-color_decorative-emerald);
   color: var(--vtmn-semantic-color_content-primary);
 }
+
+/* SIZES */
+
+/* Small */
```
Comment (Tlahey) [inline packages/sources/css/src/components/indicators/tag/src/index.css:100]: on the class `.vtmn-tag` you can remove this lines : 

```css
min-block-size: rem(24px);
block-size: rem(24px);
font-size: var(--vtmn-typo_text-3-font-size);
```

It's a duplicate of `.vtmn-tag_size--medium` :)

File: packages/sources/svelte/src/components/overlays/VtmnAlert/VtmnAlertItem.svelte [hunk 2]
```diff
@@ -50,7 +50,7 @@
   );
 </script>
 
-<div class={componentClass} role="alert" tabindex="-1" {...$$restProps}>
+<div class={componentClass} role="alert" aria-live="assertive" tabindex="-1" {...$$restProps}>
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnAlert/VtmnAlertItem.svelte:53]: Not linked to the current scope of the PR

File: packages/showcases/core/csf/components/indicators/tag.csf.js [hunk 3]
```diff
@@ -57,4 +57,13 @@ export const argTypes = {
       options: ['', ...Object.keys(vitamixIconsList)],
```
Comment (Tlahey) [inline packages/showcases/core/csf/components/indicators/tag.csf.js:57]: Please update the css storybook at `packages/showcases/css/stories/components/indicators/tag/examples`.
- Add `vtmn-tag_size--medium` on each existing stories 
- Create a new bloc with the new class like 
```html
<div class="block">
  <span class="vtmn-tag vtmn-tag_variant--accent vtmn-tag_size--small"
    >Accent Tag</span
  >

  <span
    class="vtmn-tag vtmn-tag_variant--decorative_gravel vtmn-tag_size--small"
    >Gravel</span
  >

  <a href="" class="vtmn-tag vtmn-tag_variant--alert vtmn-tag_size--small"
    ><span class="vtmx-leaf-fill" role="presentation"></span>Interactive Tag
    with icon</a
  >
</div>
```

Comment (EGilleron) [reply packages/showcases/core/csf/components/indicators/tag.csf.js:57]: Could you check again ?

File: packages/sources/svelte/src/components/indicators/VtmnTag/test/VtmnTag.spec.js [hunk 4]
```diff
@@ -88,5 +88,13 @@ describe('VtmnTag', () => {
       const { getByText } = render(VtmnTag, { href: 'http://example.com' });
       expect(getByText('Unit-test tag')).toBeVisible();
     });
+    test('Should apply size', () => {
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/indicators/VtmnTag/test/VtmnTag.spec.js:91]: Add a test by default the component should have `vtmn-tag_size--medium`

Comment (EGilleron) [reply packages/sources/svelte/src/components/indicators/VtmnTag/test/VtmnTag.spec.js:91]: done

**Review by Tlahey — APPROVED** [review]

**Review by thibault-mahe — APPROVED** [review]

**Comment by thibault-mahe** [comment]
It seems fine to me as it is. I don't add comments on some improvements already identified and planned for the v1 rewriting (class modifiers should only update CSS vars, tests should not be done on implementation details and DOM node, etc.).

Thanks a lot for you contribution! :)

**Pull request merged**